src/components/section-list.component.js

```diff
@@ -16,43 +17,45 @@ type Props = {
   buttonAction: Function,
 };
 
+const Section = styled.View`
+  margin-top: 15px;
```

Comment:
Please keep unitless for simple properties.

Reply:
I changed it to margin: 15 0 0 and this didn't work? What do you mean by unitless?

Reply:
@jamesg1 If you use unitless shorthands properties in styled components, like `margin: 15 0 0` or `margin: 15`, it will crash. But for simple properties, like `margin-top: 15`, it is OK.

I have to admit that it is a little ambiguous. Real CSS needs units, while RN doesn't need. But how about styled components in RN? For now, we prefer RN more.